.github/workflows/autopr_merged.yaml

```diff
@@ -0,0 +1,90 @@
+name: "Tag and release to production"
+
+on: 
+  pull_request:
+    types: [closed]
+
+jobs:
+  tag-release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.base.ref == 'main' && github.event.pull_request.merged && (contains(github.event.pull_request.title, '[AUTO-PR]') || contains(github.event.pull_request.title, '[MANIFEST]')) }}
```

Comment:
nit: you can get rid of the wrapping `${{ }}` here

Reply:
done and done